docs/source/about/paasta_principles.rst

```diff
@@ -14,6 +14,35 @@ for the exact contract an app must meet to run on PaaSTA is documented in the
 Principles
 ----------
 
+0. A Bespoke PaaS is not Necessarily a Bad Thing
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Building your own PaaS is a non-trivial undertaking. The decision to build your
+own, even if it makes heavy use of existing components (like PaaSTA), should
+not be taken lightly.
+
+On the plus side, building your own PaaS can give your organization more value
+than adopting an existing commercial or third-party PaaS. The value comes from
+the fact that you can build your PaaS to meet your specific needs like a glove.
+Additionally, the much of the existing engineering effort towards your current
+infrastructure can often be re-used. For example if you have an existing
+service-discovery mechanism, you can build a PaaS that integrates tightly
+with it, as opposed to replacing it with a difference service-discovery mechanism
+from a third-party.
+
+There are of course downsides to building your own PaaS. There is certainly
+some engineering effort in involved, but that must be weight against the
```

Comment:
I feel like 'some' here is a weasel word.  I would omit it in my writing; up to you.